### ISSUE TITLE
DotSpatial.WebControls is totally unusable

### ISSUE THREAD

**Issue by mogikanin** [DotSpatial/DotSpatial#951]
Do we really can say that DotSpatial "Display a map in a .NET Windows Forms or **Web** application."?
As I see DotSpatial.WebControls is totally unusable and raw. And it looks like from earlier 90s :)
My suggestion is remove it from github repository, and do not declare that DS is able to display map in Web application.

![image](https://cloud.githubusercontent.com/assets/840681/23986384/e4590a72-0a34-11e7-8ac6-9bab7978300d.png)

**Comment by jany-tenaj** [comment]
@mogikanin Yeah I know. It didn't look pretty when I tried it. Do we know enough to maybe bring it up to date? I mean there seem to be people that would like to use it.

**Comment by mogikanin** [comment]
The better is to rewrite it from scratch using latest frameworks\technologies. Not sure that current version is used by anyone. If someone wanted old version, they always can download it from codeplex repo.

**Comment by jany-tenaj** [comment]
Okay, than I'd suggest rewriting over removing.

**Comment by mogikanin** [comment]
But till we don't have rewritten version, we can't say that DS supports Web application.

**Comment by jany-tenaj** [comment]
I think removing that line for now would be alright. We can add it again after WebControls is rewritten.
I just think it would make more sense to fix the feature instead of removing it. 
That's the same I was saying about fixing Coordinate to use Z and M instead of removing them. 
I just think that if we remove everything that stopped working, some day nothing remains.

**Comment by mogikanin** [comment]
Overall, my idea is that we should not maintain such old and raw things like existing DS.WebControls.
Or e.g. look into `DotSpatial.Plugins.Taudem` It has 4 classes with 90% commented code.

**Comment by mogikanin** [comment]
Better focus on things that really used by other people, like DS.Projections, DS.Data and DS.Controls.
